neutrons/SNAPRed
# Patch restore convertdiffcallog

**Description of work**

Re-include the `ConvertDiffCalLog` SNAPred algorithm, which implemented the `OffsetMode="Signed"` functionality of the `ConvertDiffCal` mantid algorithm.  This is to meet a logic error that was overlooked in the implementation of `ConvertDiffCal`, which rejects any offsets <= -1.

**To test:**

Run the unittests, including tests of this algorithm.

Run the CIS script for testing diffraction calibration.  It should now get past the ConvertDiffCal part.



**Link to EWM item**

For defect found during [Hackathon](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=2335)
